infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -78,49 +112,54 @@ private void keepAlive(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r)
         r.sendResponse(ctx, msg, CorfuMsgType.ACK.msg());
     }
 
+    //TODO(Zach): Do we need a KEEP_ALIVE?
```

Comment:
You can remove the `KEEP_ALIVE` message type and use `PING` instead. You also need to make the corresponding changes for `NettyClientRounter::keepAlive`

Reply:
Done.